README.md

```diff
@@ -442,13 +454,11 @@ The following telemetry data is supported:
 
 - CRSF for Arduino is not compatible with AVR based microcontrollers.
   - This is because the AVR microcontrollers are simply not powerful enough to meet the minimum requirements of the CRSF protocol.
-- Certain instances of DMA can cause SAMD51 based development boards to crash.
-  - A workaround is in place to prevent this from happening, but it is not a permanent solution.
-  - This is currently being investigated.
+- The current DMA driver for both SAMD21 and SAMD51 targets have proven to be problematic and have been disabled for the time being.
```

Comment:
I really am internally debating with myself to yeet DMA altogether.  
Only SAMD21, SAMD51, and Teensy targets have DMA, and it's not imperative that the CRSF Protocol uses it.

Also, Adafruit's own ZeroDMA driver is _very_ janky. I may elect to dispatch of this later on down the line, and _maybe_ write my own DMA driver. At this point, I _really_ don't see the point in keeping DMA here. It makes no noticeable difference in terms of performance and right now, it's actually more trouble than it's worth (which is why I have kept away from it).

Reply:
Update: ZeroDMA will be removed in the next public release of CRSF for Arduino (IE After Version 0.5.0).  
Because this _has_ been a thorn in my side for quite some time, and the juice isn't worth the squeeze here.